src/jvmTest/java/org/snakeyaml/engine/usecases/references/ReferencesTest.java

```diff
@@ -95,22 +96,23 @@ public void referencesWithRecursiveKeysNotAllowedByDefault() {
   @Test
   @DisplayName("Parsing with aliases may take a lot of time, CPU and memory")
   public void parseManyAliasesForCollections() {
-    String output = createDump(25);
-    // Load
-    long time1 = System.currentTimeMillis();
-    LoadSettings settings =
+    final var minDuration = Duration.ofMillis(400);
```

Comment:
It looks very odd to me that there's a test asserting on minimum running time. I understand the upper bound, it prevents performance regressions. WDYT about leaving just the upper bound?

Reply:
Good point. I think it makes sense to verify that the test at least does something, since the result of `load.loadFromString(output)` isn't verified.

Perhaps the result can be verified - I'll have a quick look.

Aside from that, I'd prefer to keep the JVM tests as close as possible to the original SnakeJava tests, so it's easier to get updates from upstream.

Reply:
Got it. Yeah, I also noticed the result isn't verified. I also get the argument of staying close to the upstream. Let me approve and let you decide if you want to iterate on it further.

Reply:
The result is a recursive object that would be difficult to assert on! Trying to print it to console results in `java.lang.OutOfMemoryError: Java heap space`.